### PR TITLE
implement footer section, fix some issue in previous section

### PR DESCRIPTION
**Pull request by ronlechiht**
Create logo
Create contact
Create download button

### CODE REVIEW

File: practice-one/src/index.html [hunk 1]
```diff
@@ -123,6 +123,31 @@ <h3 class="card-heading"><div class="card-order">03.</div><p class="card-title">
...
+            <p class="contact-address">3, Season Park, Jakarta</p>
+            <p class="contact-email">support@foodyar.co,id</p>
+            <p class="contact-number">021 - 1111 - 2222</p>
+          </div>
+        </div>
+        <div class="footer-contact">
+          <p class="contact-title">be our friend</p>
+          <div class="contact-detail">
+            <p class="contact-address">3, Season Park, Jakarta</p>
+            <p class="contact-email">support@foodyar.co,id</p>
```
Comment (nhungduong-agilityio) [inline practice-one/src/index.html]: should you tel link and email link

File: practice-one/src/index.html [hunk 2]
```diff
@@ -123,6 +123,31 @@ <h3 class="card-heading"><div class="card-order">03.</div><p class="card-title">
 
     <!--==================================== Page Footer ===================================-->
     <footer class="footer">
+      <div class="footer-container">
```
Comment (nhungduong-agilityio) [inline practice-one/src/index.html:126]: should use ul, li for list

File: practice-one/src/index.html [hunk 3]
```diff
@@ -37,8 +37,8 @@ <h2 class="hero-title">download app, save money, make friends!</h2>
                 Footprint and help Mother Nature to sustain its beauty. So what are you waiting
                 for ? Lets ride together</p>
               <div class="hero-btn-group">
-                <button href="#" class="btn btn-secondary btn-download"><img src="assets/images/hero/chplay-ico.svg" alt="chplay">download</button>
-                <button href="#" class="btn btn-secondary btn-download"><img src="assets/images/hero/appstore-ico.svg" alt="appstore">download</button>
+                <button href="#" class="btn btn-secondary btn-download"><img src="assets/images/hero/chplay-ico.webp" alt="chplay">download</button>
+                <button href="#" class="btn btn-secondary btn-download"><img src="assets/images/hero/appstore-ico.webp" alt="appstore">download</button>
```
Comment (nhungduong-agilityio) [inline practice-one/src/index.html]: why use href for button?

File: practice-one/src/index.html (hunk 1 above)
Comment (ronlechiht) [reply practice-one/src/index.html]: i got it

File: practice-one/src/index.html (hunk 2 above)
Comment (ronlechiht) [reply practice-one/src/index.html:126]: i got it

File: practice-one/src/index.html (hunk 3 above)
Comment (ronlechiht) [reply practice-one/src/index.html]: i use <a> element first, then i change to <button> but i forget it. i will fix it

**Pull request merged**